src/component-tester.ts

```diff
@@ -151,4 +245,15 @@ export class ComponentTester<T = any> {
   }): Promise<NodeListOf<Element>> {
     return waitFor(() => this.element.querySelectorAll(selector) as NodeListOf<Element>, options);
   }
+
+  /**
+   * Register dependencies to be ignored while loading dependencies for custom element.
+   *
+   * Only Works with dependencies registered via `<require from="...">` usage
+   * Dependencies are expected to be in absolute path
+   */
+  public ignoreDependencies(...deps: string[]): this {
+    this.stubbed = deps;
```

Comment:
This does mean that multiple calls to `ignoreDependencies` override each other. I'm not sure what most people expect here, I would personally expect multiple calls to be cumulative.

```suggestion
    this.stubbed = this.stubbed.concat(deps);
```

Of course to make that work `this.stubbed` will have to be initialized to an empty array first.